src/algorithms/linear_stretching.jl

```diff
@@ -2,13 +2,17 @@
 """
 ```
     LinearStretching <: AbstractHistogramAdjustmentAlgorithm
-    LinearStretching(; minval = 0, maxval = 1)
+    LinearStretching(; [src_minval], [src_maxval], dest_minval = 0, dest_maxval = 1)
+
+    LinearStretching((src_minval, src_maxval) => (dest_minval, dest_maxval))
+    LinearStretching((src_minval, src_maxval))
```

Comment:
Instead of
```julia
 LinearStretching((src_minval, src_maxval))
```

 I would prefer if we made the API 
```julia
LinearStretching((src_minval, src_maxval) => nothing)
```
in accordance with
```julia
LinearStretching(nothing => (dst_minval, dst_maxval)).
```

My concern is that when reading code that uses the `LinearStretching` operation one may mistakenly assume that 

```julia
LinearStretching((0.2, 0.8))
```
specifies the destination interval and not the source interval, especially since the default operaton `LinearStretching()` acts on the destination interval.

Reply:
changed in b3dc554